components/NavigationBar/PlantOverviewIcon.js

```diff
@@ -8,7 +8,7 @@ export const PlantOverviewIcon = ({ getFillColor }) => (
     stroke={getFillColor}
     fill={getFillColor}
   >
-    <title>Plant Overview Icon</title>
+    <title>Plant Overview</title>
```

Comment:
you can change this into a svg file as well and set the fill and stroke directly on the Element after importing. Please update all icons in this way. The name "getFillColor" usually implies that its a function, you could use something like iconColor or fillColor
```js
import PlantOverViewIcon from './plantOverViewIcon.svg'

<PlantOverviewIcon stroke={iconColor} fill={iconColor} />